README.md

```diff
@@ -74,11 +74,53 @@ To disable, for example, the data visualization module, the following argument h
    ```
    cdk deploy  -c EnableDataviz=false
    ```
+**Note**: If using CICD deployments as described in the following step, these context variables must also be set in `refarch/aws-native/cdk.json`.
```

Comment:
I'm wondering if we want a separated section for deployment with CICD. 

For customers who are new to AWS, this step is very long and overwhelming.  I'm afraid that this step will derail them if they just want to analytics capability.

Reply:
agree to have:

- Deployment
  - Option 1: provision stack directly
  - Option 2: Use a CICD to maintain the ref arch stack

Reply:
Ok, will do.